### PR TITLE
[10.0][MIG] account_invoice_merge_payment to 10.0

### PR DESCRIPTION
**Pull request by benwillig**
This PR depends on #238

### CODE REVIEW

File: account_invoice_merge_payment/__manifest__.py [hunk 1]
```diff
@@ -1,35 +1,14 @@
...
-##############################################################################
+# Copyright 2015-2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': "account_invoice_merge_payment",
     'summary': """
         Use invoice merge regarding fields on Account Payment Partner""",
     'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
-    'website': "http://acsone.eu",
+    'website': "www.acsone.eu",
```
Comment (sbidoul) [inline account_invoice_merge_payment/__manifest__.py]: Please keep http://, it must be a valid URI

Comment (benwillig) [reply account_invoice_merge_payment/__manifest__.py]: I fixed website URI and rebased.

**Review by MiquelRForgeFlow — APPROVED** [review]
I approve because there are barely any significant changes.

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Review by ThomasBinsfeld — APPROVED** [review]
:+1:

**Pull request merged**